babel/messages/frontend.py

```diff
@@ -825,6 +882,7 @@ def _configure_command(self, cmdname, argv):
             strs = ["--%s" % name]
             if short:
                 strs.append("-%s" % short)
+            strs.extend(cmdclass.option_aliases.get(name, ()))
```

Comment:
Checking if this, combined with `strs = ["--%s" % name]`, doesn't cause any problems? If `name` is "output" you'll end up with strs being ['--output', --output']. I don't have a super solid understanding of all the different options uh, options, so for all I know this intentional. If not an issue, shipit! Everything else looks good to me.